docs/usage/shoot_versions.md

```diff
@@ -0,0 +1,108 @@
+- I can get this information via API from the CloudProfile.
+
+## Version Classifications
+
+In their lifetime, versions go through 4 distinct "logical states".
+Versions usually start with the classification `preview`, then are promoted to `supported`, eventually `deprecated` and finally `expired`. 
+This classification serves as a "point-of-reference" for end-users and also has implications during shoot creation and the maintenance time.
+
+This information is programmatically available in the `CloudProfiles` of the Garden cluster. 
+
+- **preview:** A `preview` version is a new version that has not yet undergone thorough testing, possibly a new release and needs time to be validated. There is a probability of unresolved issues and is therefore not yet recommended for production usage.
+A Shoot does not update (neither `auto-update` or `force-update`) to  a `preview` version during the maintenance time. 
+Also `preview` versions are not considered for the defaulting to the highest available version when deliberately omitting the patch version during Shoot creation.
+Typically, after a fresh release of a new Kubernetes (e.g. v1.17.0) or Machine image version (e.g. coreos-2023.5) the operator tags it as `preview` until he has gained sufficient experience and regards this version to be reliable. 
+After the operator gained sufficient trust, the version can be manually promoted to `supported`.  
+
+- **supported:** A `supported` version is the recommended version for new and existing Shoot clusters. New Shoot clusters should use and existing clusters should update to this version.
+Typically for Kubernetes versions, the latest Kubernetes patch versions of the actual (if not still in `preview`) and the last 3 minor Kubernetes versions are maintained by the community. An operator could define these versions as being `supported` (e.g. v1.16.1, v1.15.4, v1.14.9 and v1.13.12).
+
+- **deprecated:** A `deprecated` version is a version that approaches the end of its lifecycle and can contain issues which are probably resolved in a supported version. 
+New Shoots should not use this version any more. 
+Existing Shoots will be updated to a newer version if `auto-update` is enabled (`.spec.maintenance.autoUpdate.kubernetesVersion` for Kubernetes version `auto-update` or `.spec.maintenance.autoUpdate.machineImageVersion` for machine machine image version `auto-update`).
+Using automatic upgrades however does not guarantee that a Shoot runs a non-deprecated version, as the latest version (overall or of the minor version) can be deprecated as well.
+Deprecated versions should have an expiration date set for eventual expiration. 
+
+- **expired:** An `expired` versions has an expiration date in the past. 
+ New clusters with that version cannot be created and existing clusters are forcefully migrated to a higher version during the maintenance time.
+ 
+Below is an example how the relevant section of the CloudProfile might look like:
```

Comment:
Please, add a reference date with which the `expirationDate` can be compared.

Reply:
I added a reference to the Golang time package when introducing the expiration date.
The package contains extensive documentation.